client/me/profile/index.jsx

```diff
@@ -131,11 +132,11 @@ const Profile = createReactClass( {
 							>
 								{ this.state.submittingForm
 									? this.props.translate( 'Saving…' )
-									: this.props.translate( 'Save Profile Details' ) }
+									: this.props.translate( 'Save profile details' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.